spec/features/issues_spec.rb

```diff
@@ -25,30 +25,31 @@
       click_link "Edit"
     end
 
-    it "should open new issue popup" do
-      page.should have_content("Issue ##{issue.iid}")
+    it 'should open new issue popup' do
+      expect(page).to have_content("Issue ##{issue.iid}")
     end
 
-    describe "fill in" do
+    describe 'fill in' do
       before do
-        fill_in "issue_title", with: "bug 345"
-        fill_in "issue_description", with: "bug description"
+        fill_in 'issue_title', with: 'bug 345'
+        fill_in 'issue_description', with: 'bug description'
       end
 
-      it { expect { click_button "Save changes" }.to_not change {Issue.count} }
+      it { expect { click_button 'Save changes' }.
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.